docs/_samples/get_version.py

```diff
@@ -0,0 +1,14 @@
+from pylibsshext import (
+    __full_version__,  # string with both ansible-pylibssh and libssh versions
+)
+from pylibsshext import (
+    __libssh_version__,  # linked libssh lib version as a string
+)
+from pylibsshext import __version__  # ansible-pylibssh version as a string
+from pylibsshext import __version_info__  # ansible-pylibssh version as a tuple
+
+
+print(f'{__full_version__=}')
```

Comment:
This format doesn't seem to be py2 complaint. Either add a comment or change it to be py2/3 compliant.

Reply:
The comment is in the rst document for the whole page

Reply:
Oh, I need to move the box one section up: https://ansible-pylibssh--112.org.readthedocs.build/en/112/user_guide/#creating-a-ssh-session

FWIW it's not yet complete (even though there's some fixes that are already incorporated)

Reply:
Ah, scratch that. The box is in the right place, I got confused with another box...